### PR TITLE
Fix mojibake in names in copyright information

### PR DESCRIPTION
**Pull request by pabs3**
Changes-by: https://ftfy.readthedocs.io/

### CODE REVIEW

**Comment by pabs3** [comment]
Hmm, this didn't fix everything, please don't apply it yet.

**Comment by pabs3** [comment]
OK, it is ready for merging now.

File: purple2compat/COPYRIGHT [hunk 1]
```diff
@@ -424,19 +424,19 @@ Riley Patterson
...
 Ani Peter
 Luke Petre
 Diego Petten
 Nathan Peterson
 Dmitry Petroff
-SebastiÃ¡n E. Peyrott
+Sebastián E. Peyrott
 Amitakhya Phukan
 Andrea Piccinelli
 Mateusz PiÄ™kos
```
Comment (Alcaro) [inline purple2compat/COPYRIGHT:437]: Still one mojibake left

**Comment by pabs3** [comment]
Thanks, fixed.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by EionRobb** [comment]
Thanks for that. Tbh I'd forgotten that I'd added the pidgin contributors list

**Pull request merged**